view.go

```diff
@@ -470,6 +470,11 @@ func (v *View) Rewind() {
 	}
 }
 
+// IsTainted tells us if the view is tainted
+func (v *View) IsTainted() bool {
+	return v.tainted
```

Comment:
Why not expose v.tainted?

Reply:
Then they can modify it. We don't want to expose too much

Reply:
Then they can modify it. We don't want to expose the things the implementer does not need to touch

Reply:
Oww that makes cense